src/riak_kv_ts_svc.erl

```diff
@@ -490,11 +490,11 @@ sub_tsgetreq(Mod, DDL, #tsgetreq{table = Table,
             %% the columns stored in riak_object are just
             %% names; we need names with types, so:
             ColumnTypes = get_column_types(ColumnNames, Mod),
-            {reply, {tsgetresp, {ColumnNames, ColumnTypes, [Row]}}, State};
+            {reply, {tsgetresp, {ColumnNames, ColumnTypes, [list_to_tuple(Row)]}}, State};
```

Comment:
Rather than doing this here, we should ask @hmmr where he made the other changes so that rows are tuples, and do something similar.